### DISCUSSION TITLE
Problems on Pixel 7?

### DISCUSSION THREAD

**Discussion by srroof** [osmandapp/OsmAnd#15922] [General]
I just upgraded to a Pixel 7 running Android 13. When I'm in some map modes (auto, hiking but not default global), my existing Points of Interest icons multiply when the map is moved - see image. Anybody else experiencing this and have a fix?
![image](https://user-images.githubusercontent.com/56038598/206552621-f5b7f76b-ca60-4bf4-ac01-60444a93fe92.png)

**Comment by MarioSwitch** [comment]
Which version you're using?

**Comment by srroof** [comment]
It's OsmAnd 4.2.7 released 2022-07-13

--
Steve Roof
Amherst, MA 01002

**Reply by MarioSwitch** [reply]
Don't have the bug on the same version
